backend/app/middleware/request_logging.py

```
@@ -196,7 +200,7 @@ async def log_response(
                 try:
                     # Try to parse as JSON for better formatting
                     logged_response_body = json.loads(response_body)
-                except:
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)